src/services/platform.ts

```diff
@@ -12,13 +13,18 @@ async function matchTxToPlatform(tx: string): Promise<IPlatform> {
     const to = parsed.to.toLowerCase();
     const data = parsed.data.toLowerCase();
     const platforms = await Platform.find();
-
     for (const platform of platforms) {
-      for (const contract of platform.whitelist) {
-        const contractLower = contract.toLowerCase();
-        const contractNoPrefix = contractLower.substr(2);
-        if (to === contractLower || data.includes(contractNoPrefix)) {
-          return platform;
+      const platformArbitrum = platform.whitelist.ethereum[ChainId.Arbitrum];
+      const wildcard = '*';
+      if (platformArbitrum === wildcard) {
+        return platform;
+      } else {
```

Comment:
to reduce nesting, just remove the `else` since you have a `return` above so the code will work exactly the same